src/Uno.UI.RuntimeTests/Tests/Windows_UI_Xaml_Controls/Given_MediaPlayerElement.cs

```diff
@@ -22,7 +24,8 @@ public partial class Given_MediaPlayerElement
 	private static readonly Uri TestVideoUrl = new Uri("https://test-videos.co.uk/vids/bigbuckbunny/mp4/h264/720/Big_Buck_Bunny_720_10s_5MB.mp4");
 
 	[TestMethod]
-	public async void When_NotAutoPlay_Source()
+	[ActivePlatforms(Platform.Android, Platform.iOS)]
```

Comment:
```suggestion
   #if !__WASM__
   [Ignore("Not supported under wasm [ADD THE URL TO THE ISSUE]")]
   #endif
```

Reply:
i add to alll file. 
![image](https://github.com/unoplatform/uno/assets/116665025/7cfe129e-b565-4152-a953-318d7f12d2a6)
Like this one.
https://github.com/unoplatform/uno/blob/2eb2b282c0fb751ec4c964c22c152dac16c9e4bc/src/SamplesApp/SamplesApp.UITests/Microsoft_UI_Xaml_Controls/InfoBarTests/Given_InfoBar.cs#LL23C2-L23C2

Reply:
Ok, then you can remove the ActivePlatforms attribute

Reply:
Done. ActivePlatforms removed. And the links to the issues add as your sample 
[Ignore("Not supported under wasm [ADD THE URL TO THE ISSUE]")]